src/components/DashboardPage/index.jsx

```diff
@@ -30,15 +38,25 @@ const DashboardPage = ({ classes, children, title, subtitle }) => (
         {title}
         <small className={classes.subtitle}>{subtitle}</small>
       </h1>
-      <span style={{ float: 'right' }}>
-        <a href="https://github.com/mozilla-frontend-infra/firefox-health-dashboard/">
-          <img
-            src={githubMark}
-            alt="Go to Github Repo"
-            style={{ width: '2rem', padding: '0 0.5rem 0 0' }}
-          />
+      <div style={{ padding: '0 1rem 0 0', float: 'right' }}>
+        <Link to="/" title="Home">
+          <HomeIcon />
+        </Link>
+        <Link to="/android" title="Android Performance">
+          <AndroidIcon />
+        </Link>
+        <Link to="/quantum/64" title="Quantum 64 Performance">
+          <DesktopIcon />
+        </Link>
+        <Link to="/power" title="Power Usage">
+          <BatteryIcon />
+        </Link>
+        <a
+          href="https://github.com/mozilla-frontend-infra/firefox-health-dashboard/"
+          title="Source Code">
+          <CodeIcon />
```

Comment:
above code is showing icons in top left of screen:

![image](https://user-images.githubusercontent.com/2334429/58432454-6cbcea00-8080-11e9-8760-5016ab4e6f1d.png)

Reply:
Icons look nice but what does the <> mean? Perhaps you could find a clearer icon?